src/complete.js

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line import/no-cycle
+import { getTask, updateLocalStorage } from './script.js';
```

Comment:
- [OPTIONAL] It's not a good practice to disable the linters using inline-comments, I suggest removing the first line and try to fix the linter errors instead 👍

> 💡 To Fix this Linter error, please check [this link](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-cycle.md)